src/components/Button/index.examples.js

```diff
@@ -8,14 +8,13 @@ const standardButtonExamples = [
   {type: 'primary', children: 'Primary'},
   {type: 'success', children: 'Success'},
   {type: 'warning', children: 'Warning'},
-  {type: 'danger', children: 'Danger'}
+  {type: 'danger', children: 'Danger'},
+  {type: 'pill', children: 'Pill'}
```

Comment:
I think pills can come in all of the preceding types, so it might not be a "type" it either is a pill, or its not a pill.